go/runtime/registry/registry.go

```diff
@@ -195,6 +213,8 @@ type runtimeRegistry struct {
 	identity  *identity.Identity
 
 	runtimes map[common.Namespace]*runtime
+
+	storageSetCh chan struct{}
```

Comment:
This is more like "initialization completed channel"?
```suggestion
	initCh chan struct{}
```